packages/sdk/src/swap/types.ts

```diff
@@ -31,6 +31,7 @@ interface ChainsAndAssets {
 
 export interface QuoteRequest extends ChainsAndAssets {
   amount: string;
+  boostFeeBps?: number;
```

Comment:
i think this should be on `DepositAddressResponse`

Reply:
You will also be able to specify it in a quote request. We will use this param to calculate the boost fee amount and calculate the quote amount.
the `getQuote` implementation will come at a later point.

Reply:
but adding it here will add it to `QuoteResponse` which is not right 🙂 it should be in the `includedFees` there

Reply:
Ah, i see. Good point. I guess we will have to move it here once we support it inside the `getQuote` request/response. For now, i will add it to the `DepositAddressRequest`.